libs/platform/src/lib/components/link/link.component.html

```diff
@@ -0,0 +1,20 @@
+<a
+    #link
+    fd-link
+    [ngClass]="'fdp-link-truncate__txt'"
```

Comment:
You don't need [ngClass] to assign a "static" class

Reply:
converted to class